src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,10 +1,16 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private boolean hasWheels;
+    private boolean hasPlower;
+
+    public Bulldozer(boolean hasWheels, boolean hasPlower, String name, String color) {
+        this.hasWheels = hasWheels;
+        this.hasPlower = hasPlower;
+        super.setName(name);
+        super.setColor(color);
```

Comment:
Better to create constructor in Machine with two fields: name and color and here change to super(name, color) in the upper part of constructor